include/zephyr/bluetooth/mesh/keys.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_KEYS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_KEYS_H_
+
+#include <inttypes.h>
+#if defined CONFIG_BT_MESH_USES_MBEDTLS_PSA
+#include <psa/crypto.h>
+#elif defined CONFIG_BT_MESH_USES_TINYCRYPT
+#include <string.h>
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum bt_mesh_key_type {
+	ECB_KEY,
+	CCM_KEY,
+	CMAC_KEY,
+	NETWORK_KEY,
+	APPLICATION_KEY,
+	DEVICE_KEY
+};
+
+typedef uint8_t bt_mesh_raw_key_t[16];
+
+#if defined CONFIG_BT_MESH_USES_MBEDTLS_PSA
+
+#define bt_mesh_key_t psa_key_id_t
+#define INVALID_KEY_VAL  PSA_KEY_ID_NULL
+#define KEY_REF(key) &key
+
+int bt_mesh_import_key(enum bt_mesh_key_type type,
```

Comment:
This is for all these functions. Should not the action be behind the name.
```suggestion
int bt_mesh_key_import(enum bt_mesh_key_type type,
```